debian/changelog

```diff
@@ -1,16 +1,16 @@
-gatekeeper (1.0.0-0) bionic; urgency=medium
+gatekeeper (1.0.0-0) focal; urgency=medium
 
   * Rebuild for 1.0.0
 
  -- Andre Nathan <andre@digirati.com.br>  Wed, 18 Nov 2020 13:35:53 -0300
 
-gatekeeper (1.0.0~rc2-0) bionic; urgency=medium
+gatekeeper (1.0.0~rc2-0) focal; urgency=medium
 
   * Rebuild for 1.0.0-rc2
 
  -- Andre Nathan <andre@digirati.com.br>  Thu, 20 Aug 2020 10:03:55 -0300
 
-gatekeeper (1.0.0~rc1-0) bionic; urgency=medium
+gatekeeper (1.0.0~rc1-0) focal; urgency=medium
```

Comment:
Since we've released RC1 and RC2 for bionic (i.e the repository has tags for them), shouldn't we keep that way in the changelog?

Reply:
Fixed.